config/deploy.rb

```diff
@@ -26,8 +26,8 @@
       within release_path do
         execute "virtualenv", "env", "--python=/bin/python3"
         execute "./env/bin/pip3", "install", "-r", "requirements.txt"
-        execute ". /opt/xpressmp/bin/xpvars.sh && julia --project='#{release_path}/julia_envs/Xpress/' -e 'import Pkg; Pkg.instantiate(); include(\"#{release_path}/julia_envs/Xpress/build_julia_image.jl\"); build_julia_image(\"#{release_path}\")'"
         execute "./env/bin/python", "-c", "'import julia; julia.install()'"
+        execute ". /opt/xpressmp/bin/xpvars.sh && julia --project='#{release_path}/julia_envs/Xpress/' -e 'import Pkg; Pkg.instantiate(); include(\"#{release_path}/julia_envs/Xpress/build_julia_image.jl\"); build_julia_image(\"#{release_path}\")'"
```

Comment:
Yes but there is at least one file in this config directory that is still used. I think that we add cleaning up these files along with the decomposition stuff to a TODO list and merge this branch so that we can get back on track with the develop/master work flow and updating current feature branches. Are you okay with that plan?
(I put the TODOs [here](https://github.com/NREL/REopt_Lite_API/projects/1))

Reply:
I'm good with that plan.